arex-instrumentation/redis/arex-redission-v3/src/main/java/io/arex/inst/redisson/v3/RedissonInstrumentation.java

```diff
@@ -71,9 +79,9 @@ public static boolean onEnter() {
 
         @Advice.OnMethodExit(suppress = Throwable.class)
         public static <V> void onExit(@Advice.Argument(0) String name,
-            @Advice.FieldValue("commandExecutor") CommandAsyncExecutor commandExecutor,
+            @Advice.This RedissonClient redisson,
             @Advice.Return(readOnly = false) RBucket<V> redissonBucket) {
-            redissonBucket = new RedissonBucketWrapper<>(commandExecutor, name);
+            redissonBucket = new RedissonBucketWrapper<>(getCommandAsyncExecutor(redisson), name);
```

Comment:
Why not put `commandAsyncExecutorMap` in class RedissonHelper.java?

`RedissonInstrumentation` classloader is `AgentClassLoader`
`redisson` classloader is `UserClassLoader`